android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -87,6 +99,42 @@ public static void launch(Context context, HashMap<String, Object> extraData) {
         }
     }
 
+    public static Object onRequestPermissionsResult(int requestCode, Context context) {
```

Comment:
Create a PermissionsListener object instead of casting the current delegate, so this would become a local method and be much simpler.